lib/es-models/bib.js

```diff
@@ -533,6 +597,11 @@ class EsBib extends EsBase {
     }
   }
 
+  // Returns array of hashes with { url, label }, containing electronic resources that are not complete digital records
+  supplementaryContent () {
+    return this._extractElectronicResourcesFromBibMarc('supplementary content') || null
```

Comment:
Here also, empty arrays are truth, so this will never fall through to `null`. If you want a one-liner, you need something like
```
  .reduce((ret, el) => (ret || []).concat([el]), null)
```
Or maybe use a `nullIfEmpty` util?

Reply:
I see now. Thanks.